docs/usecases.md

```diff
@@ -0,0 +1,126 @@
+"Did I really give him access everywhere?"
+
+### Solution
+
+Use the [OpenID Connect authorization](reference/authorization/oidc.md).
+
+There is no need to create them somewhere on the server itself. The [OIDC authorization](reference/authorization/oidc.md) will resolve them using the configured [Identity Provider (IdP)](https://openid.net/developers/how-connect-works/) - that's it!
+
+No need to access any of these services directly to create/authorize these users.
+
+If the [environments are configured accordingly](reference/environment/index.md) (which is the default) all user's resources (like home directory) will be created automatically.
+
+## Bastion Host / Jump Host {:id=bastion}
+
+### Problem
+
+1. Assume you've to manage resources.
+2. These resources are not directly reachable for you. They are protected inside other networks, you have no direct access to. For example, you're sitting at home and there is a another service inside an [AWS private VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html).
+3. You have to manage this service.
+
+The following cases are usually used:
+
+* You need to start a VPN connection, with an VPN server to get a direct connection to this network. Either you have to deal with quirky VPN desktop client software or SSO isn't working (which might only make sense for small organizations).
+* There is a [bastion host](https://en.wikipedia.org/wiki/Bastion_host) in-place, based on [OpenSSH sshd](https://man.openbsd.org/sshd.8) which will run into [on-boarding](#onboard) and [off-boarding](#offboard) issues.
+
+### Solution
+
+1. Set up a bastion host either:
+   1. Inside the private network itself (in case of [AWS a dedicated EC2 instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html) for example of [instance-type `t2.micro`](https://aws.amazon.com/ec2/instance-types/))
+   2. or outside with a fixed VPN connection inside the private network.
+2. Configure your favored [authorization](reference/authorization/index.md) (for example [OpenID Connect](reference/authorization/oidc.md) for best [on-boarding](#onboard) and [off-boarding](#offboard) experience).
+
+## Different rules for different users per host {:id=multi-environment}
+
+### Problem
+
+1. Assume you have an SSH server.
+2. Different users should be authorized differently.
+3. Different users should be executed into different [environments](reference/environment/index.md) (the one into a local one with permission A, another one with permission B and the other one into a remote one).
+
+This is currently not really possible, except with different [OpenSSH sshd](https://man.openbsd.org/sshd.8) setups on one host, or even different hosts or hacky [PAM](https://en.wikipedia.org/wiki/Linux_PAM) or [shell](https://en.wikipedia.org/wiki/Unix_shell) setups.
+
+### Solution
+
+You're using Bifröst with multiple [flows](reference/flow.md), configured. Each flow can handle different authorization and environments.
```

Comment:
Use Bifröst with multiple [flows](reference/flow.md), configured. Each flow can handle different authorizations and environments.